longtailstorelib/gcsBlobStore_test.go

```diff
@@ -99,3 +105,96 @@ func TestGCSBlobStoreVersioning(t *testing.T) {
 		t.Errorf("object.Delete() err == %q", err)
 	}
 }
+
+func writeANumberWithRetry(number int, blobStore BlobStore) error {
+	client, err := blobStore.NewClient(context.Background())
+	if err != nil {
+		return err
+	}
+	defer client.Close()
+	object, err := client.NewObject("test.txt")
+	if err != nil {
+		return err
+	}
+	for {
+		exists, err := object.LockWriteVersion()
+		if err != nil {
+			return err
+		}
+		var sliceData []string
+		if exists {
+			data, err := object.Read()
+			if err != nil {
+				return err
+			}
+			time.Sleep(300 * time.Millisecond)
+			sliceData = strings.Split(string(data), "\n")
+		}
+		sliceData = append(sliceData, fmt.Sprintf("%05d", number))
+		sort.Strings(sliceData)
+		newData := strings.Join(sliceData, "\n")
+
+		ok, err := object.Write([]byte(newData))
+		if err != nil {
+			return err
+		}
+		if ok {
+			log.Printf("Wrote %d\n", number)
+			return nil
+		}
+		log.Printf("Retrying %d\n", number)
+	}
+}
+
+func TestGCSBlobStoreVersioningStressTest(t *testing.T) {
+	// This test uses hardcoded paths and is disabled
+	return
```

Comment:
You can and should use t.Skip() or similar mechanism here with a short mode, see https://golang.org/pkg/testing/#hdr-Skipping for an example of this